### PR TITLE
add seed testing to typescript generator

### PR DESCRIPTION
**Pull request by sahil485**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -37,6 +37,27 @@ jobs:
           name: Ensure no changes to git-tracked files
           command: git --no-pager diff --exit-code
 
+  seed-test:
```
Comment (dsinghvi) [inline .circleci/config.yml:40]: @sahil485 lets name this `seed-sdk-test` and i think we can create another task for the express docker

File: .circleci/config.yml [hunk 2]
```diff
@@ -37,6 +37,27 @@ jobs:
...
+          command: yarn workspace @fern-typescript/sdk-generator-cli dockerTagLocal:node
+      - run:
+          name: Install seed
+          command: npm install -g @fern-api/seed-cli@0.14.3-rc0-10-g57e9c3c5
+      - run:
+          name: Run seed
+          command: seed test --docker fernapi/fern-typescript-browser-sdk:local --irVersion v20 --language typescript --compile-command 'yarn install && yarn build'
+      - run:
+          name: Compare diffs
+          command: git --no-pager diff --exit-code
```
Comment (dsinghvi) [inline .circleci/config.yml:59]: shouldnt this check be erroring in ci bc none of the code is comitted?

Comment (sahil485) [reply .circleci/config.yml:59]: can u elaborate plz, not exactly sure what you mean

Comment (dsinghvi) [reply .circleci/config.yml:80]: yup what you have is perfect

File: .circleci/config.yml [hunk 3]
```diff
@@ -37,6 +37,48 @@ jobs:
...
+          pkg-manager: yarn-berry
+      - run:
+          name: Build fern-typescript-express Docker
+          command: yarn workspace @fern-typescript/express-generator-cli dockerTagLocal
+      - run:
+          name: Install seed
+          command: npm install -g @fern-api/seed-cli@0.14.3-rc0-10-g57e9c3c5
+      - run:
+          name: Run seed
+          command: seed test --docker fernapi/fern-typescript-express:local --irVersion v20 --language typescript --compile-command 'yarn install && yarn build'
```
Comment (dsinghvi) [inline .circleci/config.yml:77]: @sahil485 we should add a `--output-directory <directory>` flag to the CLI. for example, i think the sdk code snippets should live in `seed/sdk` and then the express ones in `seed/express`

Comment (sahil485) [reply .circleci/config.yml:77]: will add!

Comment (sahil485) [reply .circleci/config.yml:77]: added

**Pull request merged**